index.html

```diff
@@ -3,134 +3,401 @@
+
+     // SCALES 2
+    x2 = d3.scale.linear()
+      .domain([0, d3.max(data, function (d) { return d.MedianIncome; })])
+      .range([0, width])
+      .nice();
+    y2= d3.scale.linear()
+      .domain(d3.extent(data, function (d) { return d.rate; }))
+      .range([height, 0])
+      .nice();
+
+     
 
     // AXES
+    var padding=0
+    var axispadding=60
+    var xaxispadding=35
 
-    // EVENT HANDLERS
 
+    //Chart 1 Axes
+    var xAxis = d3.svg.axis()
+      .scale(x)
+      .orient("bottom")
+      .ticks(10);
+
+    var yAxis = d3.svg.axis()
+      .scale(y)
+      .orient("left")
+      .ticks(10);
+    
+    svg.append("g")
+      .attr("class","axis")
+      .attr("transform","translate(0,"+ height + ")")
+      .call(xAxis);
+
+    svg.append("g")
+      .attr("class","axis")
+      .attr("transform","translate("+ padding + ",0)")
+      .call(yAxis);
+
+    svg.append("text")
+      .attr("text-anchor", "middle")  // this makes it easy to centre the text as the transform is applied to the anchor
+      .attr("class","axislabel")
+      .attr("transform", "translate("+ (-axispadding) +","+(height/2)+")rotate(-90)")  // text is drawn off the screen top left, move down and out and rotate
+      .text("Arrests per 100,000 people");
+
+  svg.append("text")
+      .attr("text-anchor", "middle")  // this makes it easy to centre the text as the transform is applied to the anchor
+      .attr("class","axislabel")
+      .attr("transform", "translate("+ (width/2) +","+(height+(xaxispadding))+")")  // centre below axis
+      .text("Poverty Rate (%)");
+
+    //chart2 Axes
+
+    var xAxis2 = d3.svg.axis()
+      .scale(x2)
+      .orient("bottom")
+      .ticks(10);
+
+    var yAxis2 = d3.svg.axis()
+      .scale(y2)
+      .orient("left")
+      .ticks(10);
+    
+    svg2.append("g")
+      .attr("class","axis")
+      .attr("transform","translate(0,"+ height + ")")
+      .call(xAxis2);
+
+    svg2.append("g")
+      .attr("class","axis")
+      .attr("transform","translate("+ padding + ",0)")
+      .call(yAxis2);
+
+    svg2.append("text")
+      .attr("text-anchor", "middle")  // this makes it easy to centre the text as the transform is applied to the anchor
+      .attr("class","axislabel")
+      .attr("transform", "translate("+ (-axispadding) +","+(height/2)+")rotate(-90)")  // text is drawn off the screen top left, move down and out and rotate
+      .text("Arrests per 100,000 people");
+
+    svg2.append("text")
+      .attr("text-anchor", "middle")  // this makes it easy to centre the text as the transform is applied to the anchor
+      .attr("class","axislabel")
+      .attr("transform", "translate("+ (width/2) +","+(height+(xaxispadding))+")")  // centre below axis
+      .text("Median Income");
+
+    // EVENT HANDLERS
     d3.select('#year-range').on('change', function () {
-      options.yearRange = d3.event.target.value;
+      options.state = d3.event.target.value;
+      options.region = 'alll';
```

Comment:
I'm amused that this typo is used consistently. Probably would be better to correct, though, lest you forget and have a bug down the line.